### PR TITLE
setting a google font as the main font so it works across browsers/OS better

### PR DESCRIPTION
**Pull request by gazzdingo**
### Features and Changes

- Closes **(add link to issue here)**

### CODE REVIEW

File: packages/front-end/components/Layout/Layout.module.scss [hunk 1]
```diff
@@ -15,7 +15,7 @@ $topbarheight: 60px;
   color: #fff;
   overflow: hidden;
   overflow-y: auto;
-  font-family: Poppins, sans-serif;
+  font-family: Inter, sans-serif;
```
Comment (jdorn) [inline packages/front-end/components/Layout/Layout.module.scss:18]: If we're no longer using Poppins anywhere, can we remove the import in `global.scss`?

Comment (gazzdingo) [reply packages/front-end/components/Layout/Layout.module.scss:18]: i think we can remove poppins i wanted to double check that the Kari
but yes we can remove it if we are not going to use it.

Comment (gazzdingo) [reply packages/front-end/components/Layout/Layout.module.scss:18]: it can be removed :)

File: packages/front-end/styles/global.scss [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
 @import "bootstrap/scss/bootstrap.scss";
 @import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,300;0,400;0,500;0,600;0,700;1,400&display=swap");
```
Comment (gazzdingo) [inline packages/front-end/styles/global.scss:8]:
```suggestion
```

**Review by jdorn — APPROVED** [review]

**Comment by jdorn** [comment]
Fixed in #2692 in a different way (using `next/font`)

**Pull request closed without merging**